requirements.txt

```diff
@@ -47,3 +47,4 @@ django-hstore==1.3.5
 cognitiveatlas
 django-datatables-view
 https://github.com/benkonrath/django-guardian/archive/django-polymorphic.zip
+django-oauth-toolkit==0.9.0
```

Comment:
I don't want any surprises when the newer package version is released. Do I miss some NeuroVault guidelines for dependencies?